### DISCUSSION TITLE
Export selected drivers (only for old drivers)

### DISCUSSION THREAD

**Discussion by McDiesel8** [lostindark/DriverStoreExplorer#248] [Ideas]
When choosing for "Export Driver List" I expected to see wich drivers are old drivers or an option for "Export selected drivers" wich contains a column for driverpath to the *.inf file.

I would like this feature because there is no command line option to delete old drivers. When I can export the old drivers with path to the *.inf location I can create a script that PNPutil deletes these drivers.

I hope this feature can be added to this great creation!

**Comment by lostindark** [comment]
Added export selected drivers in new beta version.
